tests/test_test_s3_delta.py

```diff
@@ -0,0 +1,58 @@
+import logging
+import os
+import glob
+
+import pytest
+from testcontainers.minio import MinioContainer
+
+from datacontract.data_contract import DataContract
+
+logging.basicConfig(level=logging.DEBUG, force=True)
+
+datacontract = "fixtures/s3-delta/datacontract.yaml"
+file_name = "fixtures/s3-delta/data/orders.delta"
+bucket_name = "test-bucket"
+s3_access_key = "test-access"
+s3_secret_access_key = "test-secret"
+
+
+@pytest.fixture(scope="session")
+def minio_container():
+    with MinioContainer(
+        image="quay.io/minio/minio",
+        access_key=s3_access_key,
+        secret_key=s3_secret_access_key,
+    ) as minio_container:
+        yield minio_container
+
+
+def test_test_s3_delta(minio_container, monkeypatch):
+    monkeypatch.setenv("DATACONTRACT_S3_ACCESS_KEY_ID", s3_access_key)
+    monkeypatch.setenv("DATACONTRACT_S3_SECRET_ACCESS_KEY", s3_secret_access_key)
+    data_contract_str = _prepare_s3_files(minio_container)
+    data_contract = DataContract(
+        data_contract_str=data_contract_str, schema_location="fixtures/lint/custom_datacontract.schema.json"
```

Comment:
you should be able to remove the schema_location, now

Reply:
Great, thank you! Seems to work now.